content/10.diagnostics.md

```diff
@@ -168,7 +155,53 @@ Sensitivity and specificity metrics were obtained by comparison with results fro
 The estimated cost per test is about 2 euros when RNA extraction is included.
 The main strength of this test over RT-PCR is that it can be done isothermally, but the main drawback is that it is about 10-fold less sensitive than RT-PCR.
 
-##### CRISPR-based Detection
+#### Immunoassays for the Detection of Antigens
+
+Immunoassays can detect molecular indicators of SARS-CoV-2 infection, such as the proteins that act as antigens from the SARS-CoV-2 virus.
+<!-- Move this: Since 2020, many immunoassays have been developed for COVID-19 [@url:https://www.centerforhealthsecurity.org/covid-19TestingToolkit/serology/Serology-based-tests-for-COVID-19.html] using several different technologies.-->
+They offer the advantage of generally being faster and requiring less specialized equipment than other molecular tests, especially those involving PCR.
+As a result, immunoassays hold particular interest for implementation at home and in situations where resources for PCR testing are limited.
+The trade-off is that these tests typically have a lower sensitivity, and sometimes a lower selectivity, than other molecular tests.
+Immunoassays for the detection of the SARS-CoV-2 antigen can include LFTs and ELISA, as discussed here, as well as chemiluminescence immunoassays (CLIA) and chromatographic immunoassays [@doi:10.3389/fmed.2021.797109], as described in the serological testing section below.
+
```

Comment:
Another important point I think is that the time frame in which antigen test are able to detect an ongoing infection is narrower (only during peak viral load). Ref: https://www.nejm.org/doi/full/10.1056/NEJMcp2117115

Reply:
Awesome article! I'll add references to it in a few other places too! Thank you @nilswellhausen!

Reply:
I agree with that, I would plug in the work of Michael Mina here doi:10.1056/NEJMp2025631

Reply:
Fantastic!! Thank you @RLordan!